src/app/components/assets/BrainLogo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function BrainLogo(): JSX.Element {
+  return (
+    <svg
+      width="30"
+      height="30"
+      viewBox="0 0 30 30"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
Why not pass props, you might want to use the logo at a different size later :)

Reply:
can i adjust svg size with css?

Reply:
done